values.yaml

```diff
@@ -158,6 +160,9 @@ ingress:
 # TODO: Alternatively as part of reorganizing Helm values, consider moving values to existing `st2` and `st2web` sections ? (#14)
 secrets:
   st2:
+    st2chatops:
```

Comment:
Any reason this is `secrets.st2.st2chatops` instead of `secrets.st2chatops` (i.e. why the nesting under `st2`)?

Reply:
Also, any reason not to put the secrets specific to st2chatops under st2chatops (and the same for all other services) instead of aggregating all secrets to a central place?

i.e. instead of 
```
secrets:
  st2:
    st2chatops:
      foo: bar
```
do
```
st2chatops:
  secrets:
    foo: bar
```

Reply:
> Any reason this is `secrets.st2.st2chatops` instead of `secrets.st2chatops` (i.e. why the nesting under `st2`)?

For this part, not particular reason, it's true that it's a bit too verbose.

> Also, any reason not to put the secrets specific to st2chatops under st2chatops (and the same for all other services) instead of aggregating all secrets to a central place?

See issue #14 :)

Reply:
Makes sense.  I think it'd be preferable to spread the secrets out into their respective services, but we can tackle that when we tackle #14 

Thanks!

Reply:
I agree that `secrets.st2.st2chatops` looks a bit too verbose and even counter-intuitive.

This also raises some inconsistencies in the context of https://github.com/StackStorm/stackstorm-ha/issues/119 discussion. I'm thinking about exploring the alternatives for the Helm values definition.